desktop/containerd/how-to.md

```diff
@@ -0,0 +1,179 @@
+toc_min: 1
+toc_max: 2  
+---
+
+This page presents instructions about how to activate the new containerd feature for image and file system management in the Docker Engine.
+
+> **Warning**
+>
+> The containerd image store management feature is currently experimental. This may change or be removed from future releases.
+{: .warning }
+
+## Enable or disable containerd image store
+The experimental containerd image store feature is switched off by default. To change your settings:
+1. Navigate to Settings, or Preferences if you’re a Mac user.
+2. Select the Experimental features tab.
+3. Next to Use containerd for pulling and storing images, select or clear the checkbox to set your desired state.
+
+![containerd_feature_activation](images/containerd_feature_activation.png){:width="750px"}
+
+## Simple use
+Show the server version and the storage driver:
+
+```
+% docker info
+Client:
+ Context:    default
+ Debug Mode: false
+ Plugins:
+  buildx: Docker Buildx (Docker Inc., v0.9.1)
+  compose: Docker Compose (Docker Inc., v2.10.2)
+  extension: Manages Docker extensions (Docker Inc., v0.2.9)
+  sbom: View the packaged-based Software Bill Of Materials (SBOM) for an image (Anchore Inc., 0.6.0)
+  scan: Docker Scan (Docker Inc., v0.19.0)
+
+Server:
+ Containers: 0
+  Running: 0
+  Paused: 0
+  Stopped: 0
+ Images: 0
+ Server Version: 22.06.0-beta.0-372-gd3bb8227ce.m
+ Storage Driver: stargz
+  driver-type: io.containerd.snapshotter.v1
+ Logging Driver: json-file
+ Cgroup Driver: cgroupfs
+ Cgroup Version: 2
+ Plugins:
+  Volume: local
+  Network: bridge host ipvlan macvlan null overlay
+  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
+ Swarm: inactive
+ Runtimes: runc io.containerd.runc.v2
+ Default Runtime: runc
+ Init Binary: docker-init
+ containerd version: 9cd3357b7fd7218e4aec3eae239db1f68a5a6ec6
+ runc version: v1.1.4-0-g5fd4c4d
+```
+
+```
+init version: de40ad0
+ Security Options:
+  seccomp
+   Profile: builtin
+  cgroupns
+ Kernel Version: 5.10.124-linuxkit
+ Operating System: Docker Desktop
+ OSType: linux
+ Architecture: aarch64
+ CPUs: 5
+ Total Memory: 7.653GiB
+ Name: docker-desktop
+ ID: f4d28427-96df-404c-b47b-647fe5138e2a
+ Docker Root Dir: /var/lib/docker
+ Debug Mode: false
+ HTTP Proxy: http.docker.internal:3128
+ HTTPS Proxy: http.docker.internal:3128
+ No Proxy: hubproxy.docker.internal
+ Registry: https://index.docker.io/v1/
+ Labels:
+ Experimental: false
+ Insecure Registries:
+  hubproxy.docker.internal:5000
+  127.0.0.0/8
+ Live Restore Enabled: false
+```
+Run a simple container:
+```
+% docker run --rm hello-world
+Unable to find image 'hello-world:latest' locally
+7d246653d051: Download complete 
+432f982638b3: Download complete 
+46331d942d63: Download complete 
+7050e35b49f5: Downloading [>                                                  ]       0B/3.208kB
+
+Hello from Docker!
+This message shows that your installation appears to be working correctly.
+...
```

Comment:
The ellipsis on this line was to indicate I had trimmed the output.  We should format this to reflect that.

Reply:
Forgot to delete this file in my commit.